apps/api/src/services/data-provider/data-enhancer/trackinsight/trackinsight.service.ts

```diff
@@ -180,4 +163,51 @@ export class TrackinsightDataEnhancerService implements DataEnhancerInterface {
   public getTestSymbol() {
     return 'QQQ';
   }
+
+  private async searchTrackinsightSymbol({
+    requestTimeout = this.configurationService.get('REQUEST_TIMEOUT'),
```

Comment:
Just `requestTimeout,` (the assignment is not needed as it is already done in `enhance()`)